app/scripts/components/common/site-alert/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Icon } from '@trussworks/react-uswds';
+import { decode } from 'he';
+import { USWDSSiteAlert } from '$components/common/uswds/site-alert';
+
+const ALERT_KEY = 'dismissedSiteAlertUrl';
+
+function hasExpired(expiryDatetime?: Date): boolean {
+  if (!expiryDatetime) return false;
+  const expiryDate = new Date(expiryDatetime);
+  const currentDate = new Date();
+  return !!(currentDate > expiryDate);
+}
+
+enum SiteAlertType {
+  info = 'info',
+  emergency = 'emergency'
+}
+
+const infoTypeFlag = SiteAlertType.info;
+
+interface SiteAlertProps {
+  appTitle: string;
+  expires?: Date;
+  content: string;
+  type?: SiteAlertType;
+  heading?: string;
+  showIcon?: boolean;
+  slim?: boolean;
+  className?: string;
+}
+
+export default function SiteAlert({
+  appTitle,
+  expires,
+  content,
+  type = infoTypeFlag,
+  heading,
+  showIcon = true,
+  slim = false,
+  className = ''
+}: SiteAlertProps) {
+  const alertId = content;
```

Comment:
curious to why creating a copy?

Reply:
I dded it thinking that we can pass an `alertId` prop, as I feel it's better to handle different alerts like that than to rely on the content string as an unique identifier. But since the current approach works fine, I'm removing the `alertId`